frontend/public/components/sidebars/resource-sidebar-samples.tsx

```diff
@@ -26,6 +28,7 @@ import {
   referenceFor,
   referenceForModel,
 } from '../../module/k8s';
+import { catalogCategories } from '../catalog/catalog-items';
```

Comment:
I don't like this dependency. Please put the default categories in a const file.

Reply:
Code reviews 1, Auto import 0

Create catch, used the wrong import here, changed it to 

```tsx
import { defaultCatalogCategories } from '@console/dev-console/src/components/catalog/utils/default-categories';
```